ui/admin/tests/helpers/selectors.js

```diff
@@ -15,6 +15,8 @@ export const FIELD_DESCRIPTION = '[name=description]';
 export const FIELD_DESCRIPTION_VALUE = 'description';
 
 export const TABLE_RESOURCE_LINK = (url) => `tbody [href="${url}"]`;
+export const TABLE_ROW = 'tbody tr';
```

Comment:
I added these two here bc we have quite a few resources such as host sources, brokered credentials, injected credentials where this can be used.

Reply:
None blocking comment: can you point me to an example where this selector can be use? you mention the domains host sources, brokered credentials and so on. I think this selector can be useful as you mention, and don't want to miss it 😄

Reply:
ah it is for the next line basically anywhere we have this list of options such as brokered creds, injected creds we can the TABLE_ROW_CHECKBOX instead of `tbody label` that we use right now. example [here](https://github.com/hashicorp/boundary-ui/blob/89263918eadafdc8909912ba9d4b4da859c3071a/ui/admin/tests/acceptance/targets/brokered-credential-sources-test.js#L260) 

<img width="1032" alt="Screenshot 2025-03-12 at 2 15 33 PM" src="https://github.com/user-attachments/assets/20ca9bc0-aaf6-4a9c-9bb3-340f2c571827" />

Reply:
Nice addition!

Reply:
Ahh gotcha!! Thanks for the catch and the explanation, as already said, nice addition!!